settings.gradle

```diff
@@ -138,46 +138,38 @@ if (isCI) {
 
 rootProject.name = 'airbyte'
 
-// Include all java CDK modules.
-def cdkPath = rootDir.toPath().resolve('airbyte-cdk/java/airbyte-cdk')
-if (cdkPath.toFile().exists()) {
-    cdkPath.eachDir { dir ->
-        def buildFiles = file(dir).list { file, name -> name.matches("build\\.gradle(?:\\.kts)?") }
-        if (buildFiles.length == 1) {
-            def path = ":airbyte-cdk:java:airbyte-cdk:${dir.getFileName()}"
-            include path
-            project(path).name = "airbyte-cdk-${dir.getFileName()}"
-        }
-    }
+boolean hasGradleBuildFile(java.nio.file.Path path) {
+    def buildFiles = file(path).list { file, name -> name.matches("build\\.gradle(?:\\.kts)?") }
+    return buildFiles.length == 1
 }
-def bulkCdkCorePath = rootDir.toPath().resolve('airbyte-cdk/bulk/core')
-if (bulkCdkCorePath.toFile().exists()) {
-    bulkCdkCorePath.eachDir { dir ->
-        def buildFiles = file(dir).list { file, name -> name.matches("build\\.gradle(?:\\.kts)?") }
-        if (buildFiles.length == 1) {
-            def path = ":airbyte-cdk:bulk:core:${dir.getFileName()}"
-            include path
-            project(path).name = "bulk-cdk-core-${dir.getFileName()}"
+
+void includeCdkProjects(String dir, String gradlePathPrefix, String projectNamePrefix) {
+    def path = rootDir.toPath().resolve(dir)
+    if (!path.toFile().exists()) {
+        path = rootDir.toPath().resolve("airbyte-submodule").resolve(dir)
```

Comment:
This line is the only functional change in this PR and its purpose is for `includeCdkProjects` to work as expected in the airbyte-enterprise repo.